bitnami/redis-cluster/templates/scripts-configmap.yaml

```diff
@@ -76,25 +76,22 @@ data:
     fi
 {{- end }}
   ping_liveness_local.sh: |-
-    #!/bin/sh
-    set -e
+    #!/bin/bash
 
-    {{- if .Values.usePasswordFile }}
-    password_aux=`cat ${REDIS_PASSWORD_FILE}`
-    export REDISCLI_AUTH=$password_aux
-    {{- else }}
-    if [ ! -z "$REDIS_PASSWORD" ]; then export REDISCLI_AUTH=$REDIS_PASSWORD; fi;
-    {{- end }}
+    [[ -f $REDIS_PASSWORD_FILE ]] && export REDIS_PASSWORD="$(< "${REDIS_PASSWORD_FILE}")"
+    [[ -n "$REDIS_PASSWORD" ]] && export REDISCLI_AUTH="$REDIS_PASSWORD"
     response=$(
       timeout -s 3 $1 \
       redis-cli \
         -h localhost \
 {{- if .Values.tls.enabled }}
         -p $REDIS_TLS_PORT \
         --tls \
-        --cert {{ template "redis-cluster.tlsCert" . }} \
-        --key {{ template "redis-cluster.tlsCertKey" . }} \
         --cacert {{ template "redis-cluster.tlsCACert" . }} \
+        {{- if .Values.tls.authClients }}
+          --cert {{ template "redis-cluster.tlsCert" . }} \
+          --key {{ template "redis-cluster.tlsCertKey" . }} \
+        {{- end }}
```

Comment:
Please fix indentation. Args `--cert` and `--key` have two additional whitespaces.